Cargo.toml

```diff
@@ -45,6 +45,7 @@ regex = { version = "1.9", default-features = false, features = [
 
 # tree sitter
 tree-sitter-c2rust = { git = "https://github.com/liamrosenfeld/tree-sitter", branch = "cfg-fix" }
-tree-sitter-python = { git = "https://github.com/liamrosenfeld/tree-sitter-python/", branch = "wasm" }
+tree-sitter-python = { git = "https://github.com/liamrosenfeld/tree-sitter-python/", branch = "wasm"}
 tree-sitter-java = { git = "https://github.com/liamrosenfeld/tree-sitter-java/", branch = "wasm" }
 tree-sitter-c-sharp = { git = "https://github.com/liamrosenfeld/tree-sitter-c-sharp/", branch = "wasm" }
+tree-sitter-cpp = {git = "https://github.com/singh-maya/tree-sitter-cpp/", branch = "wasm"}
```

Comment:
Also missing new line here. Also, could you add the spaces around the brackets like the other lines?